### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by silverbackdan** [pouchdb-community/pouchdb-authentication#256]
The patches applied in October / November last year could do with being released to prevent a high risk vulnerability in url-parse and so we are able to upgrade without rebuilding the package in our own fork. Would it be possible to make a new release?

**Comment by assemblethis** [comment]
Yes, please, a new release.

My workaround at the moment is to include the latest in package.json as:

     "pouchdb-authentication": "pouchdb-community/pouchdb-authentication",

**Comment by MentalGear** [comment]
I agree, a new release would make sense and protect users of this package.
